src/apps/logging/logging.cpp

```diff
@@ -211,6 +248,8 @@ namespace ccfapp
         .set_result_schema(get_public_result_schema);
 
       install(Procs::LOG_RECORD_PREFIX_CERT, log_record_prefix_cert, Write);
+      install(Procs::LOG_RECORD_ANONYMOUS_CALLER, log_record_anonymous, Write)
+        .set_caller_auth_disabled(true);
```

Comment:
I think set_caller_auth(false) would be a bit clearer, but maybe that's just me.

Reply:
Absolutely.